src/test/java/net/datafaker/script/ProvidersDocsGenerator.java

```diff
@@ -170,7 +170,7 @@ private static String generateBaseHeaderRow(char padSymbol, Function<Integer, St
 
         public static String generateRow(char padSymbol, String clazzName, String javaDocComment, String groupName) {
             StringBuilder sb = new StringBuilder();
-            sb.append("|").append(generateColumn(getName(clazzName), padSymbol, NAME.length)).append("|");
+            sb.append("|").append(generateColumn(getName(clazzName, groupName.toLowerCase()), padSymbol, NAME.length)).append("|");
```

Comment:
```suggestion
            sb.append("|").append(generateColumn(getName(clazzName, groupName.toLowerCase(Locale.ROOT)), padSymbol, NAME.length)).append("|");
```
otherwise it could behave different in some locales e.g. Turkey

Reply:
but Datafaker's package names are always in the English locale. Or I'm missing something? 

Like there are `base`, `food`, `movie`, `sport`, `videogame` packages, and I don't think they will be in other locales

Reply:
it has nothing to do with datafaker's locales. It's running via `net.datafaker.script.ProvidersDocsGenerator#main` so in case of `toUpperCase()`, `toLowerCase()` it uses default system locale.  So if someone sets Turkish locale as default one it will be used

Reply:
Got it